packages/nimble-components/package.json

```
@@ -100,6 +100,7 @@
     "@microsoft/fast-react-wrapper": "^0.3.20",
     "@ni/eslint-config-javascript": "^4.2.0",
     "@ni/eslint-config-typescript": "^4.2.0",
+    "@ni/jasmine-parameterized": "1.0.0",
```

Any reason not to use a looser version match as we do elsewhere?

(Note: if you apply this you might need to regenerate package-lock.json)

```suggestion
    "@ni/jasmine-parameterized": "^1.0.0",
```